src/compiler/typescript_to_rust/generate_import_symbol_table/get_symbol_table.ts

```diff
@@ -0,0 +1,79 @@
+import * as ts from 'typescript';
+import { getDeclarationFromNamespace } from './import_export_utils';
+
+export function getSymbolTable(
+    filename: string,
+    program: ts.Program
+): ts.SymbolTable | undefined {
```

Comment:
Let's embrace Opt and Match

Reply:
Let's embrace Result, Opt, Match, Variant, etc

Reply:
We decided to not embrace these things